workflow/rules/common.smk

```diff
@@ -46,8 +46,19 @@ wildcard_constraints:
 
 
 def compile_output_list(wildcards):
-    return [
-        "fusions/gene_fuse/%s_%s_gene_fuse_fusions.txt" % (sample, t)
+    files = {
+        "fusions/gene_fuse": [
```

Comment:
This one seems unnecessary as it is the input file for `gene_fuse_report`

Reply:
fixed